core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -131,6 +131,22 @@ class HttpExecutionServiceDAO(executionServiceURL: String, override val workbenc
     retry(when5xx)(() => pipeline[ExecutionServiceLabelResponse](userInfo) apply Patch(url, labels))
   }
 
+  override def getCost(id: String,
+                       workflowCostBreakdownParams: Option[WorkflowCostBreakdownParams],
+                       userInfo: UserInfo
+  ): Future[WorkflowCostBreakdown] = {
+    val uri = Uri(executionServiceURL + s"/api/workflows/v1/$id/cost")
+    workflowCostBreakdownParams
+      .map(p =>
+        Map("includeSubworkflowBreakdown" -> p.includeSubworkflowBreakdown.toString,
+            "includeTaskBreakdown" -> p.includeTaskBreakdown.toString
+        )
+      )
+      .map(p => uri.withQuery(Query(p)))
+
+    retry(when5xx)(() => pipeline[WorkflowCostBreakdown](userInfo) apply Get(uri))
```

Comment:
Will we get logging/sentries or some other notification when these calls fail?

Reply:
This will get logged, but won't otherwise alert via Sentry or GCP. This follows the [current behavior](https://github.com/broadinstitute/rawls/blob/54010c7378acaff88da1ae6276307701708a5ad0/core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala#L341-L344) when Rawls has trouble monitoring a submission. It does pose a problem because Rawls won't be able to enforce a cost cap if it's not able to check the status of a submission's workflows for an extended period of time. 

Error handling in general probably needs some thought since it's also possible that Cromwell's cost estimation won't be complete as Janet mentioned in [another comment](https://github.com/broadinstitute/rawls/pull/3034#discussion_r1763411443). Not sure how conservative we need to be, but I'm guessing pretty conservative since user spend is at stake.